bm2/bm2.c

```diff
@@ -1,6 +1,6 @@
 /* #include <cstring> */
 #include <string.h>
-#include "strucs.h"
+#include "../bm3/strucs.h"
```

Comment:
Actually the bm2 "strucs.h" is different from the bm3 "strucs.h", but it looks like I forgot to commit it. Will rectify that shortly.